Realm/Realm.Weaver/Analytics/Analytics.cs

```diff
@@ -530,5 +530,18 @@ public enum AnalyticsCollection
             DryRun,
             Full,
         }
+
+        private class ActionDriver
```

Comment:
This could be a struct and we can probably name it something more concrete - like `FeatureAnalysisResult` or something similar.